apps/core/components/searchForm/SearchForm.js

```diff
@@ -0,0 +1,224 @@
+// @flow
+
+// @flow
```

Comment:
Extra flow declaration, should be removed.

Reply:
Yes! Just did it

Reply:
No lint that catches this 😱